src/lib.rs

```diff
@@ -169,6 +169,28 @@ fn render_message(mut w: impl Write, msg: &Message, dbc: &DBC) -> Result<()> {
         )?;
         writeln!(w)?;
 
+        for signal in msg.signals().iter() {
+            let typ = signal_to_rust_type(&signal);
+            if typ != "bool" {
```

Comment:
Smallest of all nits: I'd write it as `if typ == "bool" {  continue; }` to not have the following lines inndented :)

Reply:
I find `if typ == "bool" { continue; }` harder to read, because it "breaks" control flow.

Reply:
Maybe this one?
```rust
for signal in msg
            .signals()
            .iter()
            .filter(|sig| signal_to_rust_type(&sig) != "bool")
        {
..
```